.travis.yml

```diff
@@ -1,6 +1,6 @@
+dist: xenial
 language: node_js
 node_js:
-  - "6"
   - "10"
```

Comment:
Does this mean we could now use the bigint support to improve the percentile approximation mapping?